tests/isp/src/test_rgb2.c

```diff
@@ -29,8 +29,8 @@
 #define in_size in_h * in_w * in_ch
 
 // Output image configuration
-#define out_h  96
-#define out_w  96
+#define out_h  100
```

Comment:
```suggestion
#define out_h  96
```
id like to keep 96 192 until 200 issue is not solved, should work fine

Reply:
yes and Id like to keep same input size for all of them 192 is a better point for rgb1 2 4. Also id like to insit on that sensor size and image size do not need to be the same. Id like to change rgb1 to 192 for now.

Reply:
well, then we need to regenerate our test images

Reply:
no need see here: https://github.com/xmos/lib_camera/blob/bc14a612c342f94134367cb47a9ceac7a40100b6/tests/isp/utils.py#L61